src/config.rs

```diff
@@ -33,17 +33,25 @@ impl Config {
     ///
     /// Returns the path if the configuration file is found.
     pub fn get_default_location() -> Option<PathBuf> {
+        if let Some(config_dirs) = Config::get_default_locations() {
+            for config_dir in config_dirs {
+                if config_dir.exists() {
+                    return Some(config_dir);
+                }
+            }
+        }
+        None
+    }
+
+    #[inline(always)]
```

Comment:
Just curious about why this function is marked as `inline` 🐻

Reply:
'cause it's a trivial function so I marked it as an inline to make sure that the compiler will not call it, and we'll gain some performance, I think the compiler it'll do that anyway but I like to be explicit